packages/block-library/src/post-featured-image/edit.js

```diff
@@ -179,7 +179,7 @@ function PostFeaturedImageDisplay( {
 		) : (
 			<img
 				src={ mediaUrl }
-				alt={ media.alt_text || __( 'Featured image' ) }
+				alt={ __( 'Featured image' ) }
```

Comment:
I think it makes sense to keep this one as `Featured Image`, as it will be more helpful for assistive technologies while in the editor.

Reply:
I would like to see this left alone. If all I hear is "Featured image", how am I supposed to know what that is an image of? I could see something like "Featured image: alt_text_here" but not removing the ALT text all together. It would remove too much context considering this block could be used over and over again.

Reply:
Thank you @alexstine!
Well... The `post-featured-image` block is a block that is used when building a template. That's what it was built for, and the most frequent scenario would be inside a posts loop for archives, or an individual post template at the top.
In the context of a template, the image itself is not as relevant, because it will depend on the posts - and not the template. So what I was thinking when I made this change is that "I'd like to know that in my loop, I have a featured image here. Whether that image is a flower or a squirrel depends on the queried posts and doesn't add anything to the template itself".
Since the featured image will have to be edited/changed in the post and not the template, it made more sense to me that I know my template has a featured image, instead of a yellow sunflower.

However, I do see your point... The block is not exclusively used in templates, so I'll go ahead and make the recommended change. Using `Featured image: alt_text_here` would provide enough context in all cases 👍